partner_contact_tags_in_popup/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# SPDX-FileCopyrightText: 2023 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+{
+    "name": "Partner Contact Tags in Contacts & Addresses Pop-up",
+    "summary": """
```

Comment:
A summary should fit in online so should have one doublequote before and after, not triples.

Reply:
Done.